addons/l10n_it_edi_sdicoop/models/account_edi_format.py

```diff
@@ -52,7 +52,7 @@ def _cron_receive_fattura_pa(self):
                     _logger.info('Received file badly formatted, skipping: \n %s', file)
                     continue
 
-                invoice = self.env['account.move'].create({'move_type': 'in_invoice'})
+                invoice = self.env['account.move'].create({'move_type': 'in_invoice', 'company_id': company.id})
```

Comment:
journal_id....  (company_id is a related) @dbkosky

Reply:
ok I've updated it. However the default journal is found normally using a similar search, but using the company in the env. So it would be the same to just put  
```python
 invoice = self.env['account.move'].with_company(company).create({'move_type': 'in_invoice'})
```
or 
```python
 invoice = self.env['account.move'].with_context({'default_company_id': company.id}).create({'move_type': 'in_invoice'})
```
(see https://github.com/odoo/odoo/blob/14.0/addons/account/models/account_move.py#L65)

Reply:
might be better idd